developer-docs-site/static/scripts/install_cli.py

```diff
@@ -204,7 +204,11 @@ def __init__(self, return_code: int = 0, log: Optional[str] = None):
 
 
 class Installer:
-    METADATA_URL = "https://api.github.com/repos/aptos-labs/aptos-core/releases/latest"
+    # The API returns the newest items first. Accordingly we expect the CLI release to
+    # be in the last 100 releases (the max for a single page).
+    METADATA_URL = (
+        "https://api.github.com/repos/aptos-labs/aptos-core/releases?per_page=100"
```

Comment:
Hi team,

Perhaps silly question, but why don't we seem to reference `api.github.com/repos/aptos-labs/aptos-core` on Aptos.dev?

```
➜  developer-docs-site git:(main) pwd 
/Users/claymurphy/aptos-core/developer-docs-site
➜  developer-docs-site git:(main) grep -r "api.github.com" docs                             
➜  developer-docs-site git:(main) 
```

Should we note you can find this metadata on the following page?
https://aptos.dev/guides/aptos-api

Reply:
This is the API for GitHub itself, so we can lookup information about the releases. So nah, doesn't really make sense to mention in our docs site.

Reply:
Ack. Thanks for that, Daniel. We should update our CLI script docs to note the self updating function, shouldn't we? Or might the users simply expect that? Thanks again.

Reply:
Yeah good idea, people might not know otherwise how to update their CLI.

Reply:
Cool! Please see:
https://github.com/aptos-labs/aptos-core/pull/6423

Thanks, Daniel!